api/tests/Feature/Forms/FormStatTest.php

```diff
@@ -65,3 +65,37 @@
                 ->etc();
         });
 });
+
+
+it('checks form stats details', function () {
+    $user = $this->actingAsProUser();
+    $workspace = $this->createUserWorkspace($user);
+    $form = $this->createForm($user, $workspace, []);
+
+    // Create form submissions with varying completion times
+    $form->submissions()->createMany([
+        ['completion_time' => 60],  // 1 minute
+        ['completion_time' => 60],  // 1 minute
+        ['completion_time' => 60],  // 1 minute
+        ['completion_time' => 120], // 2 minutes
+        ['completion_time' => 120], // 2 minutes
+        ['completion_time' => null] // Incomplete submission
```

Comment:
Worth adding one without even the `completion_time` key so `[]` to ensure it doesn't break the thing